src/Assets/AssetContainer.php

```diff
@@ -139,6 +140,7 @@ public function url()
         }
 
         $url = rtrim($this->disk()->url('/'), '/');
+        $url = Str::removeLeft($url, config('app.url'));
```

Comment:
Super duper minor but would tidy this up even more, why `trim` and not removeright? And why 2 lines?

Again, super minor though.

Reply:
Imho two lines are more readable than everything nested but that's probably a matter of taste. I've replaced the `rtrim()` with `Str:replaceRight()`. Technically it's not quite the same, `rtrim` removes multiple chars at the end but `removeRight` just one. But that's probably no issue here.